ivy_tests/test_ivy/test_frontends/test_numpy/test_linear_algebra/test_matrix_eigenvalues.py

```diff
@@ -0,0 +1,35 @@
+# global
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+from ivy_tests.test_ivy.test_functional.test_core.test_linalg import (
+    _get_dtype_and_matrix,
+)
+
+
+# eigvalsh
+@handle_frontend_test(
```

Comment:
I think this is an issue with _get_dtype_and_matrix(symmetric=True) since this was passing previously and it seems to be an issue with the examples generated by the code.  In addition, ivy/ivy_tests/test_functional/test_core/test_linalg.py has the same error with the test_eigvalsh function which uses the same _get_dtype_and_matrix(symmetric=True) call.  I will continue looking into this.

Reply:
Hi @ashraggarwal, this hypothesis failed health check issue should be resolved now. Please pull from master to get the updates 😊

Reply:
I've pulled the updates and committed again.  It's showing that it passes the tests on my device.  Please review my pull request at your earliest convenience.